### ISSUE TITLE
Server crash on startup due to ClientPlayerEntity

### ISSUE THREAD

**Issue by DaringShepard** [baileyholl/Ars-Nouveau#284]
Hey there, while using the version 1.18.6 of the mod produces a fatal error on load. The error begins on line 3669 (or just search for fatal, you'll have to scroll down a bit unless you just download it. Google oddities): https://drive.google.com/file/d/1S_i2ZgrWj640TMCR4GFow2RsuA2xrKsI/view?usp=sharing

It should be stated that I'm using Mohist (version 646 https://github.com/MohistMC), but I don't believe that this is the primary cause of the issue since other mods, such as Quark, have experienced similar problems: https://github.com/Vazkii/Quark/issues/2802

Here's what I found in the log:
``[15:27:14] [main/WARN] [mixin/]: Error loading class: net/minecraft/client/entity/player/ClientPlayerEntity (java.lang.RuntimeException: Attempted to load class net/minecraft/client/entity/player/ClientPlayerEntity for invalid dist DEDICATED_SERVER)``
And
``@Mixin target net.minecraft.client.entity.player.ClientPlayerEntity was not found ars_nouveau.mixins.json:ElytraClientPlayer``

In a nutshell, I believe the mod's trying to load ClientPlayerEntity for the dedicated server, which it's not capable of doing.

Thanks in advance,
-Shepard

Edit: Corrected the report to be more accurate, as the mod was not the source of the crashing I was experiencing, that's Sodium Reforged's fault, entirely (though it *does* seem to only crash when the two mods are loaded at once). This mod, however, is still producing the fatal errors, so I'll keep the issue open, yes.

**Comment by baileyholl** [comment]
Hello,
I cannot reproduce this on my local forge server. I am not sure what Mohist is, but if it is anything like Bukkit, they will come with their own set of weird incompatibilities.  If removing Sodium fixes this for you, then that is likely an issue with that mod.

Wish I could be more help, sorry!
